### PR TITLE
BUGFIX: Select editor unset button styling

### PR DESCRIPTION
**Pull request by aertmann**

### CODE REVIEW

**Comment by aertmann** [comment]
image
image

**Comment by aertmann** [comment]
:+1:

File: TYPO3.Neos/Resources/Private/Styles/Shared/_Select2.scss [hunk 1]
```diff
@@ -198,7 +197,7 @@
 
 			i {
 				position: absolute;
-				left: 14px;
+				left: 13px;
```
Comment (gerhard-boden) [inline TYPO3.Neos/Resources/Private/Styles/Shared/_Select2.scss]: in my Chrome browser on Windows `left: 15px` seems to dead center the `x` inside the button

**Comment by gerhard-boden** [comment]
:+1: better looking hover effect inside inspector select boxes verified for:
FF 43.0.1 (Windows)
Chrome 47.0.2526.106 (Windows)
Internet Explorer 11.0.26 (Windows

**Comment by aertmann** [comment]
good catch, same for me.. thanks for reviewing

**Comment by gerhard-boden** [comment]
You're welcome!
Just out of curiosity:
Did you encounter any issues with your change why you chose to close the PR in the end?

**Comment by aertmann** [comment]
heh wrong button :D

**Pull request merged**